src/app/app.service.ts

```diff
@@ -117,6 +125,7 @@ export class AppService {
     sessionStorage.clear();
     this.userSessionData = new Object();
     this.isUserLoggedIn = false;
+    this.loggedInUserInfo = null;
```

Comment:
Why this line? @sriramj19

Reply:
Even post logout the `loggedInUserInfo` remains intact with data of the previously logged in persona. To clear that post logout, I have added it in the block where the session information are cleared.